src/fides/api/util/logger.py

```diff
@@ -42,77 +41,54 @@ def _log_warning(exc: BaseException, dev_mode: bool = False) -> None:
         logger.error(exc)
 
 
-class FidesAPIHandler(logging.Handler):
```

Comment:
this class felt overly complicated to me...why do with classes what can be done with functions? No magic overriding of the handler function, just building things transparently and adding them to `loguru`